backend/server.js

```diff
@@ -85,39 +88,38 @@ app.get("/search", async (req, res)=>{
     })
 })
 //top-tracks
-app.get("/top-tracks", async (req, res) => {
```

Comment:
top-tracks needs to be changed to previous version what you pushed is different than the last merge, Moge implemented short medium and long term, let's keep the old one

Reply:
this update broke the dashboard component not because it was not working because the end point name was changed, and in Dashboard.js we are still calling top-track instead of tracks. We can update the dashboard.js after